proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -276,9 +276,12 @@ service TableService {
 
 message TableReference {
   oneof ref {
-    // an exported ticket
+    // A ticket to resolve to get the table. To handle races correctly, we encourage this to be an export ticket, but
+    // technically any ticket type will suffice.
```

Comment:
```suggestion
    // A ticket to resolve to get the table. It's preferable to use export tickets in order to avoid races that are possible with tickets controlled by the server, but any ticket type will suffice as long as it resolves to a table.
```